README_fr_FR.md

```diff
@@ -0,0 +1,79 @@
+
+- [Spectre](https://spectreattack.com/spectre.pdf)
+- [Meltdown](https://meltdownattack.com/meltdown.pdf)
+- [DMA Remapping](https://docs.microsoft.com/en-us/windows/security/information-protection/kernel-dma-protection-for-thunderbolt)
+- [(P) ATMFD Exploit](https://msrc.microsoft.com/update-guide/en-US/vulnerability/CVE-2020-1020)
+- [(P) Print Nightmare](https://us-cert.cisa.gov/ncas/current-activity/2021/06/30/printnightmare-critical-windows-print-spooler-vulnerability)
+- [Remote Desktop](https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=Windows+Remote+Desktop)
+- [NetBIOS](https://en.wikipedia.org/wiki/NetBIOS) (*Possible Information Retrieval*)
+
+### **Nettoyé**
+
+Atlas est éxtrêmement réduit à l'éssentiel, les applications préinstallées sont enlevées, et d'autres composantes du système sont enlevées. Même si cela peut briser certaines applications, ça réduit éxtrêmement le ISO et la grandeur d'installation. Les fonctionnalités comme Windows Defender, et ceux-ci sont enlevées complètement. Cette modification est concentrée sur purement le jeu, mais la majorité des applications de travail et d'éducation marchent. [Regardez quoi d'autre nous avons enlevées dans notre FAQ (En Anglais)](https://github.com/Atlas-OS/Atlas/wiki/1.-FAQ#13-whats-removed-in-atlas-os).
+
+### **Performant**
+
+Atlas est optimisé pré-installation. En gardant la compatibilité, mais aussi s'aspire pour la performance. Nous avons epongé chaque goutté de performance dans nos ISO Windows. Quelques changements que nous avons appliqué sont ci-dessous.
+
+- Schéma d'alimentation personnalisé 
+- Services minimisés 
+- Pilotes réduits 
+- Appareils inutiles désactivés 
+- Économies d'énergie désactivées 
+- Désactivation des mesures d'atténuation de la sécurité gourmandes en performances 
+- Mode MSI activé automatiquement 
+- Optimisation de la configuration de démarrage 
+- Planification optimisée des processus
+## Kit d'art
+Vous voulez créer votre propre fond d'écran Atlas? Peut-être jouer avec notre logo pour créers votre propre design? Nous avons ça accessible au public pour allumer de l'inspiration créative autour de la communauté[Regardez notre Kit d'art et créez quelque-chose d'époustouflant.](./img/brand-kit.zip)
+
+Nous avons aussi [une séctions dédiée dans l'onglet Communauté](https://github.com/Atlas-OS/Atlas/discussions/categories/community-artwork), pour que vous pourrez partager votre design avec d'autres artistes at créer de l'inspiration.
+
+## Disclaimer
+
+By downloading, modifying, or utilizing any of these images, you agree to [Microsoft's Terms.](https://www.microsoft.com/en-us/Useterms/Retail/Windows/10/UseTerms_Retail_Windows_10_English.htm) None of these images are pre-activated, you **must** use a genuine key.
```

Comment:
Should this line also be translated?

Reply:
I mean the chinese translation doesn't have it, so consistency?

Reply:
> I mean the chinese translation doesn't have it, so consistency?

Just noticed that. Weird, but yeah, consistency.